apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog/GoogleStackdriverTarget_Configuration.cs

```diff
@@ -117,11 +117,18 @@ public partial class GoogleStackdriverTarget
 
         /// <summary>
         /// Fills <see cref="LogEntry.JsonPayload"/> instead of <see cref="LogEntry.TextPayload"/>.
-        /// JSON serialization can be customed using <see cref="JsonConverterTypeName"/> and <see cref="JsonConverterMethodName"/>,
+        /// JSON values serialization can be customized using <see cref="JsonConverterTypeName"/> and <see cref="JsonConverterMethodName"/>,
         /// or <see cref="JsonConverter"/>.
         /// </summary>
         public bool SendJsonPayload { get; set; }
 
+        /// <summary>
+        /// Fills <see cref="LogEntry.JsonPayload"/> in specified by user way with <see cref="T:NLog.Layouts.JsonLayout"/> layout type.
```

Comment:
```suggestion
        /// Fills <see cref="LogEntry.JsonPayload"/> in a manner specified by the <see cref="NLog.Layouts.JsonLayout"/> layout type.
```

(The "T:" can confuse VS and is recommended against by FxCop. The other change is just to make the English make more sense - please let me know if I've misinterpreted the intention.)

Reply:
I'm using IDEA Rider and it 'sees' reference after adding T:.

Reply:
Well, VS doesn't - so please change it. (I happened to make the same change in NodaTime this weekend.)

Reply:
Hmm. It didn't work.
https://github.com/googleapis/google-cloud-dotnet/pull/5635/checks?check_run_id=1479470031#step:7:14
It breaks build, so I'm going to change that back

Reply:
@amanda-tarafa I was sure that my link will refer to history, but I can't find it now. Build failed because of that. I can turn it back but it'll fail again.